pkg/visor/rpc.go

```diff
@@ -9,6 +9,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/SkycoinProject/skywire-mainnet/pkg/app/launcher"
```

Comment:
Not sorted.

Reply:
@nkryuchkov We need to find out whether we can make `goimports` do this automatically for us, or we need to write our own `go generate` tool.

Reply:
@evanlinjin Yeah, sorting imports is a pain 😔

Reply:
https://github.com/SkycoinProject/skywire-mainnet/issues/361